src/modules/transaction/utils/transaction.js

```diff
@@ -93,7 +92,7 @@ const convertObjectToHex = (data) => {
   // eslint-disable-next-line no-restricted-syntax, no-unused-vars, guard-for-in
   for (const key in data) {
     const value = data[key];
-    if (key === 'votes' || key === 'unlockObjects') {
+    if (key === 'stakes') {
```

Comment:
was there a reason excluding ` key === 'unlockObjects'`

Reply:
Previously we needed to send the stakes to unlock (unlockObject) when unlocking stakes. In the new SDK version we should not send back the schema in the transaction hex. As can be seen in the schema documentation for `pos:unlock`
<img width="517" alt="image" src="https://user-images.githubusercontent.com/8784876/214271155-6a08cd28-f169-4e16-bbb9-bea7a1abfb18.png">

Good to know, before the new SDK update all module commands needed a schema. Now we have 2 module commands which shouldn't pass along a schema to the transaction (`pos:unlock` and `pos:claimRewards`). One cannot unlock specific stakes and rewards, one can only unlock **all** unlocks for ones address.